compiler/natives/src/net/http/fetch.go

```diff
@@ -116,7 +117,7 @@ func (t *fetchTransport) RoundTrip(req *Request) (*Response, error) {
 		},
 		func(reason *js.Object) {
 			select {
-			case errCh <- errors.New("net/http: fetch() failed"):
+			case errCh <- fmt.Errorf("net/http: fetch() failed: %s", reason.String()):
```

Comment:
Can you be sure that `reason` will never be nil? Otherwise it'd be a nil pointer dereference panic, right?